lib/rule-tester/rule-tester.js

```diff
@@ -96,6 +99,19 @@ const { SourceCode } = require("../source-code");
  * @property {boolean} [only] Run only this test case or the subset of test cases with this property.
  */
 
+/**
+ * A test case that is expected to trigger a fatal error / exception.
+ * @typedef {Object} FatalTestCase
+ * @property {string} [name] Name for the test case.
+ * @property {string} [code] Code for the test case.
+ * @property {TestCaseFatalError} error Expected error/exception.
+ * @property {any[]} [options] Options for the test case.
+ * @property {{ [name: string]: any }} [settings] Settings for the test case.
+ * @property {string} [filename] The fake filename for the test case. Useful for rules that make assertion about filenames.
+ * @property {LanguageOptions} [languageOptions] The language options to use in the test case.
```

Comment:
```suggestion
 * @property {string} [parser] The absolute path for the parser.
 * @property {{ [name: string]: any }} [parserOptions] Options for the parser.
 * @property {{ [name: string]: "readonly" | "writable" | "off" }} [globals] The additional global variables.
 * @property {{ [name: string]: boolean }} [env] Environments for the test case.
```

rule-tester expects eslintrc config options.